pkg/cmd/util/resource.go

```diff
@@ -0,0 +1,80 @@
+package util
```

Comment:
We generally shouldn't be adding these things to cmd/util - it has nothing to do with those.  Instead, extract it within pkg/cmd/cli/cmd/* as non public methods.